app/services/dttp/import_user.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Dttp
+  class ImportUser
+    include ServicePattern
+
+    def initialize(user_hash:)
+      @user_hash = user_hash
+    end
+
+    def call
+      user.update!(attributes)
+    end
+
+  private
+
+    attr_reader :user_hash
+
+    def attributes
+      {
+        first_name: user_hash["firstname"],
+        last_name: user_hash["lastname"],
+        email_address: user_hash["emailaddress1"],
+        dttp_id: contact_id,
+        provider_id: user_hash["_parentcustomerid_value"],
```

Comment:
```suggestion
        provider_dttp_id: user_hash["_parentcustomerid_value"],
```

Probably less misleading